v2/pkg/runner/validate.go

```diff
@@ -34,6 +34,10 @@ func (options *Options) validateOptions() error {
 		return errors.New("hostip flag must be used with RemoveWildcard option")
 	}
 
+	// Match or Filter are not allowed together
```

Comment:
Similarly to `httpx`, I think the tool should support both matching and filtering, sequentially [applying them](https://github.com/projectdiscovery/httpx/blob/e4552498194fd46eca2b8984aad9cc6d49b3cb58/runner/runner.go#L575)

Reply:
Done